.github/workflows/deploy-sepolia-vaults-keeper-helper.yaml

```
@@ -0,0 +1,40 @@
+    steps:
+      - uses: actions/checkout@master
+        with:
+          fetch-depth: 0
+          submodules: recursive
+      - uses: foundry-rs/foundry-toolchain@v1
+      - run: forge install
+      - run: forge build
+      - run: forge tree
+      - name: Run deploy script
+        run: forge script script/TermVaultsKeeper.s.sol:TermVaultsKeeperDeployer --rpc-url $RPC_URL --verify --vvvv --broadcast
+        environment:
```

actionlint (1.7.4)

unexpected key "environment" for "step" section. expected one of "continue-on-error", "env", "id", "if", "name", "run", "shell", "timeout-minutes", "uses", "with", "working-directory"

(syntax-check)